apps/frontend/src/modules/projects/hooks/useTheme.ts

```diff
@@ -19,16 +20,17 @@ export type ColorScheme = {
 };
 
 export const useThemes = () => {
-    const { user } = useUserState();
     const { workspace } = useWorkspace();
     const [createThemeMutation] = useProjectThemeCreateMutation();
     const [updateThemeMutation] = useProjectThemeUpdateMutation();
     const { updateSurvey } = useSurvey();
```

Comment:
This will eventually create a cyclic dependency. It will make this hook hard to test later. Why do we need `useSurvey` hook here anyway? We should probably make two calls whenever we create/delete a theme.

Reply:
@kaf-lamed-beyt I think we should move the `updateSurvey` to the function the theme is created. The reason is that we should be able to create a theme without a survey. A quick example, in project settings, we'll have a manage themes feature. Updating survey in this hook will break this feature.